unittests/Feature/FeatureModelTransaction.cpp

```diff
@@ -904,4 +904,128 @@ TEST_F(FeatureModelTransactionTest, removeFeatureFromModel) {
   EXPECT_FALSE(FM->getFeature("ab"));
 }
 
+TEST_F(FeatureModelTransactionTest, removeFeaturesFromModel) {
+  size_t FMSizeBefore = FM->size();
+
+  // Prepare Model with several Features
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ab"),
+                            FM->getFeature("a"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ac"),
+                            FM->getFeature("a"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ad"),
+                            FM->getFeature("ab"));
+  vara::feature::addFeature(*FM, std::make_unique<BinaryFeature>("ae"),
+                            FM->getFeature("ad"));
+
+  // Prepare Features for deletion
+  std::vector<detail::FeatureVariantTy> FeaturesToBeDeleted;
+  FeaturesToBeDeleted.emplace_back(FM->getFeature("ad"));
+  FeaturesToBeDeleted.emplace_back(FM->getFeature("ac"));
+  FeaturesToBeDeleted.emplace_back(FM->getFeature("ae"));
```

Comment:
```suggestion
  std::vector<detail::FeatureVariantTy> FeaturesToBeDeleted{
  FM->getFeature("ac"), FM->getFeature("ad"), FM->getFeature("ae")
  };
```
This should also work. (- formatting)